### PR TITLE
Fix getElementBoundingBox() OOP version returning 6 numbers instead of a 2 vectors

### PR DESCRIPTION
**Pull request by Pirulax**
Small'n'Easy to test.
Fixes the OOP version of getElementBoudingBox.
1. Now it returns 2 vectors if MinClientReqCheck passed
2. Now it's available as a class variable too

### CODE REVIEW

**Comment by Pirulax** [comment]
Also, please see the **header** file, I left a comment there, please help me out with that :)

File: Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.h
Comment (patrikjuvonen) [inline Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.h]: This is ok. You should remove `LUA_DECLARE(GetElementBoundingBox)` as `LUA_DECLARE_OOP(GetElementBoundingBox)` automatically does both. This duplicate declaration causes [a compile error](https://ci.appveyor.com/project/multitheftauto/mtasa-blue/build/ci-1603#L4208) C2535.

**Review by patrikjuvonen — CHANGES_REQUESTED** [review]

File: Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.h
Comment (Pirulax) [reply Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.h]: Thanks, I'll remove it.

File: Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.h
Comment (patrikjuvonen) [inline Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.h]: No, you did it the other way around :)

Remove `LUA_DECLARE(GetElementBoundingBox)`, keep `LUA_DECLARE_OOP(GetElementBoundingBox)`.

For your reference, this is how they work:
```cpp
#define LUA_DECLARE(x) static int x ( lua_State * luaVM );
#define LUA_DECLARE_OOP(x) LUA_DECLARE(x) LUA_DECLARE(OOP_##x)
```

**Review by patrikjuvonen — CHANGES_REQUESTED** [review]

**Comment by Addlibs** [comment]
I think the 510990e (min_mta_version tag in meta) solution is more elegant than a bool argument for legacy style

File: Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.h
Comment (Pirulax) [reply Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.h]: Ahh, alright, thanks :)

**Review by patrikjuvonen — APPROVED** [review]

**Comment by patrikjuvonen** [comment]
Reminder for committer: `MIN_CLIENT_REQ_GETBOUNDINGBOX_OOP` needs to be bumped before merge.

**Comment by qaisjp** [comment]
Thank you!

**Pull request merged**
